app/src/main/kotlin/com/exallium/todoapp/screenbundle/ScreenBundleHelper.kt

```diff
@@ -10,4 +10,6 @@ interface ScreenBundleHelper {
     fun setTitle(bundle: Bundle, @StringRes screenTitleRes: Int)
     fun setTitle(bundle: Bundle, screenTitle: String)
     fun getTitle(bundle: Bundle?): String
+    fun setNoteId(bundle: Bundle, noteId: String? = "")
```

Comment:
let this be null by default. We're allowed to accept null here as it represents the absence of a note.